alchemist/alchemist-implementationbase/src/main/java/it/unibo/alchemist/model/implementations/environments/Continuous2DObstacles.java

```diff
@@ -150,12 +162,13 @@ private void writeObject(final ObjectOutputStream o) throws IOException {
         o.writeObject(getObstacles());
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
@DanySK here

Reply:
What if you cast to RectObstacle2D<?>?

Reply:
It complains because `rtree` should contain `RectObstacle2D<Euclidean2DPosition>`

Reply:
Meh, leave the suppression there then